### DISCUSSION TITLE
Creating custom coil profiles

### DISCUSSION THREAD

**Discussion by Moby1971** [shimming-toolbox/shimming-toolbox#427] [Q&A] [answered]
Dear authors,

Thank you for the excellent paper and making this toolbox publicly available.

Question: 
Is there a manual, instruction, or example of how to create a custom B0 coil profile ?
Like described in section 2.1.3 of your paper ?

many thanks
Gustav Strijkers

**Comment by po09i** [comment]
Hi Gustav,

Thank for you interest! We currently do not have a tutorial on creating custom coil profiles. However, this is something I am currently working on (https://github.com/shimming-toolbox/shimming-toolbox/pull/312). I will update you once this is available. This should be available by the end of next week.

Alex

**Comment by po09i** [comment, marked as answer]
Hello @Moby1971,

I have made this [tutorial](https://shimming-toolbox-py--312.org.readthedocs.build/en/312/user_section/tutorials/create_b0_coil_profiles.html) that shows how to create a custom B0 coil profile. You can do the tutorial by updating to the latest version of Shimming Toolbox.

```
git clone https://github.com/shimming-toolbox/shimming-toolbox.git
cd shimming-toolbox
make install
```

Alex
